docs/concepts/package-types.md

```diff
@@ -0,0 +1,225 @@
+| `esm-view` | Yes             | Yes             | Yes            | Yes            | No                         | Yes + rewriting external dependencies to CDN | The compiled bundle will use `src/index.tsx` as an entrypoint and [link it](../esm-views/output-package-manifest.md) in the output `package.json`; the entrypoint needs to default export a React component for `modular start`/ [standalone `index.html`](../esm-views/how-to-build.md) to work |
+| `package`  | No              | Yes             | Yes            | Yes            | N/A                        | No                                           | Whatever specified in `main` field of `package.json` is used as entrypoint                                                                                                                                                                                                                       |
+| `view`     | Yes             | Yes             | Yes            | Yes            | N/A                        | No                                           | Whatever specified in `main` field of `package.json` is used as entrypoint; needs to default export React component for `modular start` to work                                                                                                                                                  |
+| `source`   | No              | No              | Yes            | Yes            | N/A                        | N/A - never built                            | N/A - don't need the concept of entrypoint since this package is never built                                                                                                                                                                                                                     |
+| `template` | No              | No              | Yes            | Yes            | Depends on the target type | N/A - never built                            | N/A - don't need the concept of entrypoint since this is a special type of package used by `modular add` which is never built directly                                                                                                                                                           |
+
+## App
+
+Modular `app`s are Typescript and React based Single Page Applications (SPA)
+built with [Webpack v5](https://webpack.js.org/) (by default) or
+[esbuild](https://esbuild.github.io/) (by turning on `useModularEsbuild` in
+[the configuration](../configuration.md)).
+
+Apps are generated by `modular add` using the
```

Comment:
I think this could be 2 sentences to read a bit easier.

The 2nd sentence could start like this:

> ... templates and need at least a ...

becomes

> ... templates. They need at least ...

Reply:
Split the sentence and moved the template part to the bottom, like all the other types.